.github/workflows/continuous-integration.yml

```diff
@@ -324,7 +324,20 @@ jobs:
       matrix:
         buildtype: [vagovdev, vagovstaging, vagovprod]
 
-    needs: [unit-tests, contract-tests, security-audit, nightwatch]
+    needs: [nightwatch, unit-tests, contract-tests, security-audit, linting]
```

Comment:
Everything before this is just a rearrangement of jobs. The main changes are below.